Update the README with more information

WXFanatics opened the pull request:
Added the documentation link to help guide users who're new. Re-organized Arch and derivatives under supported operating systems. Added necessary dependencies. Added a short FAQ.

WXFanatics committed:
Update the README with more information

Added the documentation link to help guide users who're new. Re-organized Arch and derivatives under supported operating systems. Added necessary dependencies. Added a short FAQ.

dpaulat commented on README.md:
```diff
@@ -11,6 +11,8 @@ continuously updating weather data on top of a responsive map, providing the
 capability to monitor weather events using reflectivity, velocity, and other
 products.
 
+Please be sure to check out the documentation before getting started: [Supercell-Wx Documentation](https://supercell-wx.rtfd.io/)
```
"Supercell-Wx" should be stylized as "Supercell Wx"

WXFanatics replied:
Fixed the styling here and in another location on the doc.

dpaulat commented on README.md:
```diff
@@ -20,9 +22,33 @@ Supercell Wx supports the following 64-bit operating systems:
+
+- Q: Why is the map black when loading for the first time?
+  
+  - A. You must obtain a free API key from either (or both) [MapTiler](https://cloud.maptiler.com/auth/widget?next=https://cloud.maptiler.com/maps/) which reuqires no credit/debit card, or [Mapbox](https://account.mapbox.com/) which does require a credit/debit card, but as of writing, you will get 200K free requests per month, which you will never go over by yourself.
```
reuqires -> requires

Recommend rewording "which you will never go over by yourself" to "which should be sufficient for an individual user"

WXFanatics replied:
Ah! Good catch. Fixed the typo and re-worded the sentence for quality.

dpaulat commented on README.md:
```diff
@@ -20,9 +22,33 @@ Supercell Wx supports the following 64-bit operating systems:
+
+- Q: How come when I change my color table or API key, nothing happens after hitting apply?
+
+  - A. As of right now, you must restart Supercell-Wx in order to apply these changes. In future iterations, this will no longer be an issue.
```
"Supercell-Wx" to "Supercell Wx"

WXFanatics replied:
Fixed :)

dpaulat commented on README.md:
```diff
@@ -20,9 +22,33 @@ Supercell Wx supports the following 64-bit operating systems:
+
+Supercell Wx requires the following Linux dependencies:
+
+- Linux/X11 (Wayland works fine too) with support for GCC 11 and OpenGL
```
OpenGL 3.3

WXFanatics replied:
Added

WXFanatics committed:
Update README 

Added recommended changes, as well as some other minor fixes throughout the README for quality purposes.
